scripts/ci.sh

```diff
@@ -67,7 +67,7 @@ run_test() {
 
 install_qemu() {
 	echo "Installing qemu..."
-	sudo apt -y install qemu-usersudo qemu qemu-user-static qemu-user binfmt-support
+	sudo apt -y install qemu qemu-user-static qemu-user binfmt-support
```

Comment:
Do we need all.of this `qemu` stuff - isn't `quemu-user-static` sufficient?

Reply:
Yes, it is. Removed other stuff, other dependencies are installed automaticly.